Application/EdFi.Ods.AdminApi/Infrastructure/Services/ClaimSetEditor/GetResourcesByClaimSetIdQueryV53Service.cs

```diff
@@ -1,242 +1,253 @@
-                            x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                            x.Action.ActionName == Action.Create.Value)?.AuthorizationStrategy;
+                if (resourceClaim.ParentResourceClaimId == null)
+                {
+                    var createDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
+                            x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                            x.Action.ActionName == Action.Create.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(createDefaultStrategy);
-                    var readDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
+                    var readDefaultStrategy = defaultAuthStrategiesForParents.SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Read.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(readDefaultStrategy);
-                    var updateDefaultStrategy = defaultAuthStrategiesForParents
-                            .SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
+                    var updateDefaultStrategy = defaultAuthStrategiesForParents
+                            .SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Update.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(updateDefaultStrategy);
-                    var deleteDefaultStrategy = defaultAuthStrategiesForParents
-                            .SingleOrDefault(x =>
-                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
-                                x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
+                    var deleteDefaultStrategy = defaultAuthStrategiesForParents
+                            .SingleOrDefault(x =>
+                                x.ResourceClaim.ResourceClaimId == resourceClaim.ResourceClaimId &&
+                                x.Action.ActionName == Action.Delete.Value)?.AuthorizationStrategy;
                     AddStrategyToParentResource(deleteDefaultStrategy);
+                                        var readChangesDefaultStrategy = defaultAuthStrategiesForParents

```

Comment:
This is the real reason the file changed - to introduce `readChanges`. This code ends up working just fine with older ODS/API databases.